.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,101 @@
+name: PHP Coding Standards
+
+on:
+    push:
```

Comment:
Not sure if there is a different convention in use in this repo, but the "standard" convention for Yaml files is indents of 2 spaces.

Reply:
It seems that other .yaml files in this repo use four spaces indents.
E.g., https://github.com/WordPress/gutenberg/blob/trunk/.github/workflows/publish-npm-packages.yml